app/src/main/res/layout/view_suggested_edits_card.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <org.wikipedia.feed.view.CardHeaderView
+        android:id="@+id/headerView"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content" />
+
+    <androidx.viewpager2.widget.ViewPager2
+        android:id="@+id/seCardsPager"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content" />
+
+    <com.google.android.material.tabs.TabLayout
```

Comment:
The indicators should be clickable.

Reply:
Robin and I discussed this, and since they can't be farther apart to make it clickable and have a ripple, we decided to have them just as indicators and not clickable.. same with news card and otd events.

Reply:
I think the current news card supports it. #1665

Reply:
Whoops! missed this! Will add here too.